lib/segment/src/vector_storage/dense/appendable_dense_vector_storage.rs

```diff
@@ -124,11 +130,12 @@ impl<T: PrimitiveVectorElement, S: ChunkedVectorStorage<T>> VectorStorage
         stopped: &AtomicBool,
     ) -> OperationResult<Range<PointOffsetType>> {
         let start_index = self.vectors.len() as PointOffsetType;
+        let disposed_hw = HardwareCounterCell::disposable(); // This function is only used for operations.
```

Comment:
Should it be like this?

```suggestion
        let disposed_hw = HardwareCounterCell::disposable(); // This function is only used during optimization.
```

Reply:
It should've been "internal operations". Fixed